tests/Doctrine/ODM/MongoDB/Tests/Functional/VersionTest.php

```diff
@@ -3,9 +3,19 @@
 namespace Doctrine\ODM\MongoDB\Tests\Functional;
 
 use Doctrine\ODM\MongoDB\Mapping\Annotations as ODM;
+use Documents\Phonenumber;
 
 class VersionTest extends \Doctrine\ODM\MongoDB\Tests\BaseTest
 {
+    public function tearDown()
+    {
+        global $concurrentRequestLogic;
+        if (isset($concurrentRequestLogic)) {
+            unset($concurrentRequestLogic);
+        }
+        unset($concurrentRequestLogic);
```

Comment:
Should I bother fixing this? Something tells me you don't want random global closures in the codebase.